src/Chart/Radar.php

```diff
@@ -535,7 +536,7 @@ public function drawRadar(Image $Object, Data $Values, array $Format = [])
                     $Points[count($Points) - 1][1],
                     $Points[0][0],
                     $Points[0][1],
-                    $Color
+                    $Color + (isset($Format['LineWeight']) ? [ 'Weight' => $Format['LineWeight'] ] : [] ),
```

Comment:
Please remove the trailing `,`, it is not supported on older PHP versions.

Reply:
Done